<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDIMainWindow</class>
 <widget class="QMainWindow" name="MDIMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sACNView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSpinBox {
border: 1px solid palette(mid); 
height: 22px;
}

QSpinBox:focus {
border: 1px solid palette(highlight); 
}

QSpinBox:disabled{
border: 1px solid palette(dark);
}

QSpinBox::up-button {
subcontrol-position: right;
width: 22px;
height: 22px;
background-color:palette(button);
border-image: url(:/icons/spin_up_dark.png) 1;
}

 QSpinBox::up-button:disabled {
background-color:grey;
 }

QSpinBox::down-button {
subcontrol-position: left;
width: 22px;
height: 22px;
background-color:palette(button);
border-image: url(:/icons/spin_down_dark.png) 1;
}

QSpinBox::down-button:disabled {
background-color:grey;
 }</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScopeView"/>
   <addaction name="actionRecieve"/>
   <addaction name="separator"/>
   <addaction name="actionTranmsit"/>
   <addaction name="actionSnapshot"/>
   <addaction name="actionMultiUniverse"/>
   <addaction name="actionPCAPPlayback"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QDockWidget" name="dwUnivList">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Universe List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>All</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Start at Universe</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QToolButton" name="btnUnivListBack">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res/resources.qrc">
                <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbUniverseList">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnUnivListForward">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res/resources.qrc">
                <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="treeView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="itemsExpandable">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item>
            <widget class="QPushButton" name="pbFewer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>(-) Show Fewer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbMore">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Show More (+)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDiscovered">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Discovered</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTreeView" name="treeViewDiscovered">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="itemsExpandable">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionScopeView">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/scope.png</normaloff>:/icons/scope.png</iconset>
   </property>
   <property name="text">
    <string>ScopeView</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opens an oscilloscope style view of the levels of a set of addresses over time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/snapshot.png</normaloff>:/icons/snapshot.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opens a window allowing snapshot and playback of a universe of sACN levels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionTranmsit">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/transmit.png</normaloff>:/icons/transmit.png</iconset>
   </property>
   <property name="text">
    <string>Tranmsit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opens a window allowing transmission of a universe of sACN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionRecieve">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/univ_view.png</normaloff>:/icons/univ_view.png</iconset>
   </property>
   <property name="text">
    <string>Recieve</string>
   </property>
   <property name="toolTip">
    <string>View a universe of sACN</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Configure application settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About sACNView</string>
   </property>
  </action>
  <action name="actionMultiUniverse">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/multi_universe.png</normaloff>:/icons/multi_universe.png</iconset>
   </property>
   <property name="text">
    <string>Send Multiple Universes</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open a window allowing easy setup of multiple sources of sACN data at once&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionPCAPPlayback">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/capture_playback.png</normaloff>:/icons/capture_playback.png</iconset>
   </property>
   <property name="text">
    <string>Playback</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Playback data captured with Wireshark (or other tools that produce a PCAP file format). sACN Data will be played back with the same timing as when it was captured, allowing precise recreation of fades.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
